es6-arrow-functions/main.js

```diff
@@ -0,0 +1,46 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+  tellJoke: (setup, punchline) => {
+    $jokeForm.classList.add('d-none');
+    const $introStatement = jokester.renderJokePhrase('Hey Flash...');
+    jokester.appendJokePhrase($introStatement);
+    setTimeout(function () {
+      const $jokeSetup = jokester.renderJokePhrase(setup);
```

Comment:
While this works it is bad practice to refer directly to the object `jokester`, this should be using `this`.